setup.py

```diff
@@ -138,7 +142,7 @@ def build_extension(self, ext):
             preset = ("windows-cl" if platform.system() == "Windows" else platform.system().lower()) + suffix
         _log_and_run(
             cmake,
-            "-DTEST=NO",
+            f"-DTEST={int(ARTICDB_BUILD_CPP_TESTS)}",
```

Comment:
so this will eval to `true` / `false` but the docs say:
```
True is any expression evaluating to: 1, ON, TRUE, YES, and Y.
False is any expression evaluating to: 0, OFF, FALSE, NO, N, IGNORE, and NOTFOUND.
```
https://enccs.github.io/cmake-workshop/cmake-syntax/#flow-control

Reply:
This came with https://github.com/man-group/ArcticDB/pull/486#discussion_r1232357608 which proposes using `ON` and `OFF` directly.

Reply:
If you read the [official docs](https://cmake.org/cmake/help/latest/command/if.html), it says:

> Named boolean constants are case-insensitive.